examples/hello/timeBP/timeBPWriter.cpp

```diff
@@ -48,13 +48,13 @@ int main(int argc, char *argv[])
     {
         // Define variable and local size
         adios::Variable<double> &ioMyDoubles =
-            adios.DefineVariable<double>("myDoubles", {Nx});
+            adios.DefineLocalArray<double>("myDoubles", false, {Nx});
 
         adios::Variable<float> &ioMyMatrix =
-            adios.DefineVariable<float>("myMatrix", {rows, columns});
+            adios.DefineLocalArray<float>("myMatrix", true, {rows, columns});
 
         adios::Variable<float> &ioMyMatrix2 =
-            adios.DefineVariable<float>("myMatrix2", {rows, columns});
+            adios.DefineLocalArray<float>("myMatrix2", false, {rows, columns});
```

Comment:
See my comment below about localArray......from looking at it I think I am expecting-->  float array[rows][columns];